src/linear_algebra/affine_space/affine_map.lean

```diff
@@ -497,6 +497,18 @@ instance : module k (P1 →ᵃ[k] V2) :=
 
 @[simp] lemma coe_smul (c : k) (f : P1 →ᵃ[k] V2) : ⇑(c • f) = c • f := rfl
 
+@[simp] lemma smul_linear (t : k) (f : P1 →ᵃ[k] V2) : (t • f).linear = t • f.linear := rfl
+
+/-- The space of affine maps between two modules is linearly equivalent to the product of the
+domain with the space of linear maps. -/
```

Comment:
```suggestion
domain with the space of linear maps, by taking the value of the affine map at `(0 : V1)` and the linear part. -/
```
Or something; this is data, so you shouldn't just say "are equivalent", but should describe the actual equivalence you chose.

Reply:
Thanks, done.